KPP/fullchem/fullchem.kpp

```diff
@@ -1,11 +1,12 @@
-#MINVERSION   2.5.0       { Need this version of KPP or later          }
-#INTEGRATOR   rosenbrock  { Use Rosenbrock integration method          }
-#LANGUAGE     Fortran90   { Generate solver code in Fortran90 ...      }
-#UPPERCASEF90 on          { ... with .F90 suffix (instead of .f90)     }
-#DRIVER       none        { Do not create gckpp_Main.F90               }
-#HESSIAN      off         { Do not create the Hessian matrix           }
-#MEX          off         { MEX is for Matlab, so skip it              }
-#STOICMAT     off         { Do not create stoichiometric matrix        }
+#MINVERSION   2.5.0                  { Need this version of KPP or later          }
```

Comment:
MINVERSION should be 2.6.0 or greater, as that is when the KPP adaptive solver came in.